### PR TITLE
Bugfix, Tooltips and minimap indicator

### PR DESCRIPTION
**Pull request by sp00ktober**
So far contains:

- Added Tooltips to the Nebula ingame settings
- Added fix and setting for error `System.ObjectDisposedException` on host resulting in random client disconnect
- Added code to handle rare IndexOutOfBounds in `PlanetFactory.Import()`
- Fix wrong array size in ILS storage and slot
- Added remote player minimap indicator
- Add command to navigate to planet, star or player
- Add command to list planets in a solar system
- Listing players now also displays their id

grafik
grafik
grafik

### CODE REVIEW

**Review by PhantomGamers — CHANGES_REQUESTED** [review]
If you trigger a tooltip to pop up and then close the menu, the tooltip persists on the screen indefinitely 
image

Also I think the /system command should use the system the player is currently in (if they are in one) if it is used without any arguments.

Other than those two things this looks good!

**Review by PhantomGamers — APPROVED** [review]
LGTM

File: NebulaWorld/Chat/Commands/NavigateCommandHandler.cs [hunk 1]
```diff
@@ -88,7 +100,7 @@ public string GetDescription()
         }
         public string GetUsage()
         {
-            return "[planet name | planet id | star name | star id | clear | player] [player id | playr name]";
+            return "[planet name | planet id | star name | star id | clear]\r\nUsage: /n player [player id | playr name]";
```
Comment (kremnev8) [inline NebulaWorld/Chat/Commands/NavigateCommandHandler.cs:103]: I don’t think this solution is good. Instead change this method to return an array and change help command.

**Review by kremnev8 — CHANGES_REQUESTED** [review]

File: NebulaWorld/Chat/Commands/HelpCommandHandler.cs [hunk 2]
```diff
@@ -43,9 +43,16 @@ private static string GetCommandDetailsOutput(string commandName)
             string aliasList = ChatCommandRegistry.GetCommandAliases(commandName).Join(FullCommandName);
 
             StringBuilder sb = new StringBuilder();
+            foreach(string usage in handler.GetUsage())
+            {
+                sb.Append($"Usage: {FullCommandName(commandName)} {usage}\n");
+            }
+            string usageString = sb.ToString();
```
Comment (kremnev8) [inline NebulaWorld/Chat/Commands/HelpCommandHandler.cs:50]: This is a weird way of doing this. Move iterations down and save one string variable allocated.

**Review by kremnev8 — CHANGES_REQUESTED** [review]

**Pull request merged**